src/components/bookList.css

```diff
@@ -0,0 +1,147 @@
+.book-container {
```

Comment:
- [ ] kindly consider using the same font in the original design so your app looks identical to the zeplin design

![Screenshot_20220716_100419](https://user-images.githubusercontent.com/25227022/179348376-ecc73b6a-3308-4959-8b8b-817b634d0bf5.png)